### DISCUSSION TITLE
Is ReadWriteMany Reliable?

### DISCUSSION THREAD

**Discussion by hutger** [longhorn/longhorn#4448] [Q&A]
Hi There, 

I'm looking for a StorageClass that provides `ReadWriteMany` mode and LongHorn seems to be a good candidate. However, as per the docs for the latest version (1.3.1), `ReadWriteMany` is still an experimental feature. 

Question: Is `ReadWriteMany` mode reliable? If it's not, would you recommend a different tool to that end?

**Comment by derekbit** [comment]
It's still an experiment feature in v1.3. We plan to make it GA in v1.4.
For now, you can try it and give us feekbacks.

**Reply by hutger** [reply]
Hi @derekbit , awesome. Is there an ETA for v1.4?

**Reply by innobead** [reply]
The current plan is the next January. ref: https://github.com/longhorn/longhorn/milestone/33
